### ISSUE TITLE
p-values instead of z-scores

### ISSUE THREAD

**Issue by basel-maher** [xqwen/integrative#10]
is it possible to use enloc with p-values from GWAS, instead of z-scores? The GWAS I would like to use has only provided p-values.

Thanks

**Comment by xqwen** [comment]
Yes, you can convert it to z-score by ``qnorm(pval/2)`` in R. Your converted z-scores are all negative, but the computation only uses z^2 information. 

Very soon, we will add the functionalities to do this in our new implementation of enloc.
